src/state/trajectory/logics.ts

```diff
@@ -4,38 +4,41 @@ import queryString from "query-string";
 // NOTE: importing @aics/simularium-viewer here currently breaks ability to compile in testing setup
 // TODO: work on test babel setup, or switch to jest?
```

Comment:
The above comments are very intriguing :)

Reply:
Yeahhh I think a long time ago I tried switching (partially?) to Jest to see if it fixed anything per @meganrm 's suggestion but it didn't make a difference. Maybe the import will work now? haha